pkg/apis/openebs/v1alpha1/jivavolumepolicy_types.go

```diff
@@ -49,7 +51,7 @@ type JivaVolumePolicySpec struct {
 // TargetSpec represents configuration related to jiva target deployment
 type TargetSpec struct {
 	// Monitor enables or disables the target exporter sidecar
-	Monitor bool `json:"monitor,omitempty"`
+	Monitor bool `json:"monitor"`
```

Comment:
should it still be omitempty and use the default value as `true` if it is omitted?

Reply:
How are we going to differentiate between the case when the user inputs false and the user left it empty?
To handle this case, we have made monitor field as required.